tmd/io/io.py

```diff
@@ -103,12 +106,48 @@ def load_neuron(input_file, line_delimiter='\n', soma_type=None,
     for i in range(comp[0]):
         tree_ids = _np.where(comp[1] == i)[0] + len(soma_ids)
         tree = make_tree(data[tree_ids])
-        neuron.append_tree(tree, td=tree_types_final)
+        neuron.append_tree(tree, tree_types_final)
 
     return neuron
 
 
-def load_population(neurons, tree_types=None, name=None):
+def load_neuron_from_morphio(path_or_obj, tree_types=None):
+    """
+    Create Neuron object from morphio object or from path
+        loaded via morphio.
+        Supported file formats: h5, swc, asc.
+
+        Args:
+            path_or_obj (Union[str, morphio.Morphology]):
+                Filepath or morphio object
+
+        Returns:
+            neuron (Neuron): tmd Neuron object
+    """
+    from morphio import Morphology  # pylint: disable=C0415
```

Comment:
better write the specific error, so that we know why it complains e.g.
disable=import-outside-toplevel